assets/quick-order-list.css

```diff
@@ -208,12 +188,6 @@ quick-order-list .quantity__button {
   font-weight: 400;
 }
 
-.product-option {
-  font-size: 1.4rem;
-  word-break: break-word;
-  line-height: calc(1 + 0.5 / var(--font-body-scale));
-}
-
 .variant-item__sold-out {
```

Comment:
I think this is due to the max height which we dont need on mobile because the footer isnt sticky

https://screenshot.click/29-11-rmaty-23whg.png